crates/cheatnet/src/state.rs

```diff
@@ -317,13 +318,12 @@ impl TraceData {
     }
 }
 
-fn match_node_response<T: Default>(result: StateResult<T>) -> StateResult<T> {
+fn match_node_response<T>(result: StateResult<T>) -> StateResult<T> {
     match result {
         Ok(class_hash) => Ok(class_hash),
-        Err(StateError::StateReadError(msg)) if msg.contains("node") => {
-            Err(StateError::StateReadError(msg))
-        }
-        _ => Ok(Default::default()),
+        Err(StateReadError(msg)) if msg.contains("node") => Err(StateReadError(msg)),
```

Comment:
This arm is not needed, since the last arm would do the same anyway?

Reply:
That's right